source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -140,26 +146,31 @@ To install the VirtualBox kernel modules, here are the steps:
 
      clr-boot-manager update
 
+.. note:: 
+
+  :abbr:`LGA (Linux Guest Additions)` drivers are integrated in Clear Linux LTS kernel 4.14 going forward. You do not need to manually install LGA. 
```

Comment:
@bwarden Is the above note necessary, given the reversion to `Install Guest Additions` below?

Reply:
No, it's unnecessary.